packages/core/crates/core-crypto/src/types.rs

```diff
@@ -480,6 +480,12 @@ impl TryFrom<[u8; 32]> for Hash {
     }
 }
 
+impl Into<[u8; 32]> for Hash {
```

Comment:
Implementing `Into` is not sound, you should consider implementig `From` which gives you automatically the inverse implementation of `Into`

Reply:
changed